gnocchi/tests/functional/gabbits/aggregates-fetch.yaml

```diff
@@ -0,0 +1,147 @@
+          - ["2015-03-06T14:34:12+00:00", 1.0, 4.0]
+          - ["2015-03-06T14:34:15+00:00", 1.0, 5.0]
+          - ["2015-03-06T14:35:12+00:00", 1.0, 10.0]
+          - ["2015-03-06T14:35:15+00:00", 1.0, 15.0]
+
+    - name: get aggregates
+      POST: /v1/aggregates/fetch
+      data:
+        operations: ["metric", ["$HISTORY['create metric1'].$RESPONSE['$.id']", "mean"], ["$HISTORY['create metric2'].$RESPONSE['$.id']", "mean"]]
+      response_json_paths:
+        $.`len`: 2
+        $."$HISTORY['create metric1'].$RESPONSE['$.id']_mean":
+          - ["2015-03-06T14:33:00+00:00", 60.0, 43.1]
+          - ["2015-03-06T14:34:00+00:00", 60.0, -2.0]
+          - ["2015-03-06T14:35:00+00:00", 60.0, 10.0]
+          - ["2015-03-06T14:33:57+00:00", 1.0, 43.1]
+          - ["2015-03-06T14:34:12+00:00", 1.0, 12.0]
+          - ["2015-03-06T14:34:15+00:00", 1.0, -16.0]
+          - ["2015-03-06T14:35:12+00:00", 1.0, 9.0]
+          - ["2015-03-06T14:35:15+00:00", 1.0, 11.0]
+        $."$HISTORY['create metric2'].$RESPONSE['$.id']_mean":
+          - ["2015-03-06T14:33:00+00:00", 60.0, 2.0]
+          - ["2015-03-06T14:34:00+00:00", 60.0, 4.5]
+          - ["2015-03-06T14:35:00+00:00", 60.0, 12.5]
+          - ["2015-03-06T14:33:57+00:00", 1.0, 2.0]
+          - ["2015-03-06T14:34:12+00:00", 1.0, 4.0]
+          - ["2015-03-06T14:34:15+00:00", 1.0, 5.0]
+          - ["2015-03-06T14:35:12+00:00", 1.0, 10.0]
+          - ["2015-03-06T14:35:15+00:00", 1.0, 15.0]
+
+    - name: get aggregates granularity
+      POST: /v1/aggregates/fetch?granularity=60
+      data:
+        operations: ["metric", ["$HISTORY['create metric1'].$RESPONSE['$.id']", "max"], ["$HISTORY['create metric2'].$RESPONSE['$.id']", "min"]]
+      response_json_paths:
+        $.`len`: 2
+        $."$HISTORY['create metric1'].$RESPONSE['$.id']_max":
+          - ["2015-03-06T14:33:00+00:00", 60.0, 43.1]
+          - ["2015-03-06T14:34:00+00:00", 60.0, 12.0]
+          - ["2015-03-06T14:35:00+00:00", 60.0, 11.0]
+        $."$HISTORY['create metric2'].$RESPONSE['$.id']_min":
+          - ["2015-03-06T14:33:00+00:00", 60.0, 2.0]
+          - ["2015-03-06T14:34:00+00:00", 60.0, 4.0]
+          - ["2015-03-06T14:35:00+00:00", 60.0, 10.0]
+
+    - name: get aggregates mean aggregate
+      POST: /v1/aggregates/fetch
+      data:
+        operations: "(aggregate mean (metric ($HISTORY['create metric1'].$RESPONSE['$.id'] mean) ($HISTORY['create metric2'].$RESPONSE['$.id'] mean)))"
+      response_json_paths:
+        $.`len`: 1
+        $."aggregated":
+          - ["2015-03-06T14:33:00+00:00", 60.0, 22.55]
+          - ["2015-03-06T14:34:00+00:00", 60.0, 1.25]
+          - ["2015-03-06T14:35:00+00:00", 60.0, 11.25]
+          - ["2015-03-06T14:33:57+00:00", 1.0, 22.55]
+          - ["2015-03-06T14:34:12+00:00", 1.0, 8.0]
+          - ["2015-03-06T14:34:15+00:00", 1.0, -5.5]
+          - ["2015-03-06T14:35:12+00:00", 1.0, 9.5]
+          - ["2015-03-06T14:35:15+00:00", 1.0, 13.0]
```

Comment:
can we add back all the tests we removed here: https://github.com/gnocchixyz/gnocchi/pull/426/files

or just let me know if you think the original test was useless... or is not relevant anymore. :)

Reply:
Next update will have all missing negative tests.